k9mail/src/main/java/com/fsck/k9/preferences/Settings.java

```diff
@@ -35,7 +35,7 @@
      *
      * @see SettingsExporter
      */
-    public static final int VERSION = 44;
+    public static final int VERSION = 46;
```

Comment:
(the reason this is incremented by 2 is because the settings were moved in separate commits)